CHANGELOG.md

```diff
@@ -14,6 +15,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Deprecate API fields `Order.discount`, `Order.discountName`, `Order.translatedDiscountName` - #6874 by @korycins
 - Fix argument validation in page resolver - #6960 by @fowczarek
 - Drop `data` field from checkout line model - #6961 by @fowczarek
+- Add `PRODUCT_VARIANT_CREATED`, `UPDATED`, `DELETED` webhooks, fix attributes field for `PRODUCT_CREATED`, `UPDATED` webhooks - #6963 by @piotrgrundas
```

Comment:
My bad that I didn't explicitly  provide you a proper sample. it should be `PRODUCT_VARIANT_CREATED`, `PRODUCT_VARIANT_UPDATED`, `PRODUCT_VARIANT_DELETED`